lib/Raven/SerializerInterface.php

```diff
@@ -10,9 +10,8 @@
      * sanitization and encoding.
      *
      * @param mixed $value
-     * @param int   $max_depth
-     * @param int   $_depth
+     * @param array $context
```

Comment:
Missing doc for the argument. Plus, I would just put `mixed` as return value instead of a long list of scalar types